particles/Tutorial/Kotlin/README.md

```diff
@@ -351,11 +351,20 @@ In essence, a store is where the data physically exists. The handle lets differe
 In the last tutorial, we set the Person handle inside of the particles. Person was also a singleton, it represented a single instance of the Person schema. In this tutorial, we will instead define a collection with multiple instances of Person inside the Arcs Manifest file. While a collection can be thought of as a list, there are no guarantees about the order of elements within the collection.
 Using some more sophisticated template interpolation, we can easily greet everyone in our collection. But that’s enough explanation, let’s get to some code!
 
-We begin with the Arcs manifest file which includes the collection of `PersonDetails`.
+We begin with the Arcs manifest file which only includes the collection of `PersonDetails`.
```

Comment:
This needs some more explanation, it isn't clear that there should be 2 Arcs Manifest Files. Also, the first file contains the schema for PersonDetails. It appears the Collection is still in the main Arcs Manifest File.